src/components/Card.js

```diff
@@ -19,9 +11,8 @@ export default class Card extends Component {
 
   constructor(properties) {
     super(properties);
-    this.state = {
-      childProps: sanitizeChildProps(properties)
-    };
+    const { style, ...childProps } = properties;
+    this.childProps = childProps;
```

Comment:
If we move childProps away from state, the component will not rerender of a change in childprops. Can that not create trouble.